fastseq/optimizer/fairseq/generate_old.py

```diff
@@ -0,0 +1,418 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Optimize fairseq-generate (v0.9.0)"""
```

Comment:
This file will not be needed?

Reply:
No. It's vestigial. Will remove it on the next commit.